doubleml/utils/maketables_mixin.py

```diff
@@ -0,0 +1,195 @@
+
+        # Build coefficient table with required and optional columns
+        coef_table = pd.DataFrame(
+            {
+                "b": self.coef,  # Required: coefficient estimates
+                "se": self.se,  # Required: standard errors
+                "p": self.pval,  # Required: p-values
+                "t": self.t_stat,  # Optional: t-statistics
+                "ci95l": ci.iloc[:, 0],  # Optional: lower 95% CI bound
+                "ci95u": ci.iloc[:, 1],  # Optional: upper 95% CI bound
+            }
+        )
+
+        # Set index to match summary table (handles treatment variable names)
+        if hasattr(self, "summary") and self.summary is not None and len(self.summary) > 0:
+            coef_table.index = self.summary.index
+
+        return coef_table
+
+    def __maketables_stat__(self, key: str):
+        """
+        Return model statistic by key.
+
+        Parameters
+        ----------
+        key : str
+            The statistic key to retrieve. Common keys include:
+            - 'N': number of observations
+            - 'r2': R-squared (not applicable for DoubleML)
+            - 'adj_r2': adjusted R-squared (not applicable for DoubleML)
+            - 'aic': Akaike Information Criterion (not applicable for DoubleML)
+            - 'bic': Bayesian Information Criterion (not applicable for DoubleML)
+            - 'll': log-likelihood (not applicable for DoubleML)
+
+        Returns
+        -------
+        float, int, or None
+            The requested statistic value, or None if not available or not applicable.
+
+        Notes
+        -----
+        DoubleML focuses on causal inference, not prediction, so traditional model fit
+        statistics like R-squared, AIC, and BIC are not applicable and will return None.
+        Currently only 'N' (number of observations) is supported.
+
+        Examples
+        --------
+        >>> n_obs = dml_model.__maketables_stat__('N')
+        >>> r2 = dml_model.__maketables_stat__('r2')  # Returns None
+        """
+        stats_map = {
+            "N": self.n_obs if hasattr(self, "n_obs") else None,
+        }
+        return stats_map.get(key, None)
+
+    @property
+    def __maketables_depvar__(self) -> str:
+        """
+        Return the name of the dependent variable.
+
+        Returns
+        -------
+        str
+            Name of the dependent (outcome) variable. Defaults to "Y" if not available.
+
+        Notes
+        -----
+        Retrieves the dependent variable name from the DoubleMLData object's y_col attribute.
+        Falls back to "Y" if the attribute is not available.
+
+        Examples
+        --------
+        >>> depvar = dml_model.__maketables_depvar__
+        >>> print(depvar)
+        'Y'
+        """
+        if hasattr(self, "_dml_data") and hasattr(self._dml_data, "y_col"):
+            return self._dml_data.y_col
+        return "Y"  # Fallback
+
+    @property
+    def __maketables_default_stat_keys__(self) -> list:
```

Comment:
Would there be other model stats users should be allowed to display at the bottom of the regression table? The relevant ETable function argument would be `model_stats`:  https://py-econometrics.github.io/maketables/reference/ETable.html#maketables.ETable